lib/ansible/module_utils/ipa.py

```diff
@@ -28,13 +28,28 @@
 # USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import json
+import socket
 
 import re
 from ansible.module_utils._text import to_bytes, to_native, to_text
 from ansible.module_utils.six import PY3
 from ansible.module_utils.six.moves.urllib.parse import quote
 from ansible.module_utils.urls import fetch_url
-from ansible.module_utils.basic import env_fallback
+from ansible.module_utils.basic import env_fallback, AnsibleFallbackNotFound
+
+
+def _env_then_dns_fallback(*args, **kwargs):
```

Comment:
Is there a good reason to make this "private"? Seems as though a callback from args should be public.

Reply:
well... It's semi-private, since I only used one underscore and not two. And did it so, since this is the only module where this function is called from. If you think I should remove the leading underscore, I can remove it.